runtest.lua

```diff
@@ -522,78 +534,75 @@ local function perform_test(file, completion, env)
 end
 
 if reload_mode then
+	---@cast explicit_tests -nil
 	while true do
-		print("Loading " .. test_name)
-		evaluator.typechecker_state:speculate(function()
-			local shadowed, test_env = env:enter_block(terms.block_purity.effectful)
-			local ok, test_expr, test_env = load_alc_file(test_name, test_env, print)
+		for _, test_path in ipairs(explicit_tests) do
```

Comment:
Reload mode should not be iterating over anything, it should operate on a single file.